Stripe/STPSourceParams.m

```diff
@@ -150,17 +150,16 @@ + (STPSourceParams *)sepaDebitParamsWithName:(NSString *)name
     params.type = STPSourceTypeSEPADebit;
     params.currency = @"eur"; // SEPA Debit must always use eur
 
-    NSDictionary<NSString *,NSString *> *address =
-    @{
-      @"line1": addressLine1,
-      @"city": city,
-      @"postal_code": postalCode,
-      @"country": country
-      };
-
+    NSMutableDictionary<NSString *,NSString *> *address = [NSMutableDictionary new];
+    address[@"city"] = city;
+    address[@"postal_code"] = postalCode,
+    address[@"country"] = country;
+    if (addressLine1) {
```

Comment:
Is this check even necessary? Setting a nil value with a mutable dictionary is valid iirc.